README.md

```diff
@@ -19,37 +19,55 @@ RKE2 is a fully compliant Kubernetes distribution, with the following changes:
 
 ## Installation
 
-RKE2 comes with a convenient and configurable installation script that handles OS configuration, binary release download, and installation of an openrc init script or systemd unit file. 
+RKE2 comes with a convenient installation script that will, by default, install via `yum` on RPM-based systems while unpacking the tarball bundle to `/usr/local` on all others.
 
 ### Quick Start
 
-The `install.sh` script provides a convenient way for installing rke2 server and agent, to install rke2 service:
+To install via the default mechanism for your system:
 
 ```sh
-INSTALL_RKE2_VERSION=v0.0.1-alpha.7 ./install.sh
+# Y.O.L.O.
+curl -fsSL https://raw.githubusercontent.com/rancher/rke2/master/install.sh | sudo sh -
 ```
 
-### CIS Mode
-
-If you plan to run your RKE2 cluster in CIS mode, the installation script offers the ability to install RKE2 in preparation for this. Use the `INSTALL_RKE2_CIS_MODE` environment variable. This will configure the system for RKE2 to run in CIS compliant mode. This includes the creation of a user "etcd", updates a number of kernel parameters, and installs a systemd or openrc unit file that passes the CIS mode flag to the binary (`--profile=cis-1.5`). Please reference the installation script (`install.sh`) for CIS specifics.
```

Comment:
Is this information going to be put somewhere else? I think it has value to the user.